### PR TITLE
Enhance generate_static_provisioning_yamls.sh

### PR DESCRIPTION
**Pull request by amdabhad**
- https://github.com/IBM/ibm-spectrum-scale-csi/issues/567

Please check the type of change your PR introduces:
- [x] Bugfix
- [x] Feature Enhancement
- [ ] Test Automation 
- [ ] Code Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Community Operator listing 
- [ ] Other (please describe): 

## How risky is this change?
- [x] Small, isolated change
- [ ] Medium, requires regression testing
- [ ] Large, requires functional and regression testing

### CODE REVIEW

**Comment by amdabhad** [comment]
For directory based static PV:
VolumeHandle:      0;0;7171748422707577770;13280B0A:61F4048E;;;/ibm/fs1/amol

For dependent fileset based static PV:
VolumeHandle:      0;1;7171748422707577770;13280B0A:61F4048E;;dep1;/ibm/fs1/fset2/dep1

For independent fileset based static PV:
VolumeHandle:      0;2;7171748422707577770;13280B0A:61F4048E;;fset2;/ibm/fs1/fset2

**Comment by amdabhad** [comment]
For directory based static PV:
VolumeHandle:      0;0;7171748422707577770;13280B0A:61F4048E;;;/ibm/fs1/amol

For dependent fileset based static PV:
VolumeHandle:      0;1;7171748422707577770;13280B0A:61F4048E;;dep1;/ibm/fs1/fset2/dep1

For independent fileset based static PV:
VolumeHandle:      0;2;7171748422707577770;13280B0A:61F4048E;;fset2;/ibm/fs1/fset2

File: tools/generate_static_provisioning_yamls.sh [hunk 1]
```diff
@@ -84,7 +84,7 @@ spec:
     - ${accessmode}
   claimRef:
     name: pvc-${volname}
-    namespace: ibm-spectrum-scale-csi
+    namespace: <PVC namespace>
```
Comment (deeghuge) [inline tools/generate_static_provisioning_yamls.sh:87]: Lets not use the claimRef approach for binding pv to pvc. Remove the reference for pv instead use volumeName: <replace with pv name> in pvc

File: tools/generate_static_provisioning_yamls.sh [hunk 2]
```diff
@@ -110,7 +110,7 @@ apiVersion: v1
 kind: PersistentVolumeClaim
 metadata:
   name: pvc-${volname}
-  namespace: ibm-spectrum-scale-csi
+  namespace: <PVC namespace> 
 spec:
   accessModes:
     - ${accessmode}
```
Comment (deeghuge) [inline tools/generate_static_provisioning_yamls.sh:116]: add `volumeName: ${volname}` in pvc. Add it at same level as accesscode and resources

File: tools/generate_static_provisioning_yamls.sh (hunk 1 above)
Comment (amdabhad) [reply tools/generate_static_provisioning_yamls.sh]: Done

File: tools/generate_static_provisioning_yamls.sh (hunk 2 above)
Comment (amdabhad) [reply tools/generate_static_provisioning_yamls.sh:114]: Done

**Review by deeghuge — APPROVED** [review]

**Review by Jainbrt — APPROVED** [review]

**Pull request merged**